extensions/positron-r/extension.webpack.config.js

```diff
@@ -1,18 +1,21 @@
 /*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
  *--------------------------------------------------------------------------------------------*/
 
 //@ts-check
 
 'use strict';
 
-const path = require('path');
 const withDefaults = require('../shared.webpack.config');
 
 module.exports = withDefaults({
 	context: __dirname,
 	entry: {
 		extension: './src/extension.ts',
 	},
+	externals: {
```

Comment:
We rely on Positron itself to make this module available and won't bundle it with positron-r.